### ISSUE TITLE
Reinstate UI gauge to show CPU usage 

### ISSUE THREAD

**Issue by maks** [democloid/picoTracker#131]
While working in battery gauge I found there was already UI code from lgpt to show CPU usage on song screen during project playback but this was not functional for picotracker.

I also found that in the main loop there is pico specific code to measure CPU usage but only for debug printing to console.

So implementing this should just be a matter of wiring up reading the CPU usage out of a variable that the main loop code updates while playback is happening by reinstating the previous UI code in the song screen.

**Comment by democloid** [comment]
That CPU gauge is fundamentally broken, doesn't really work. In any case, it should only be used for development and/or debugging purposes. See comments in #8

**Comment by maks** [comment]
Yes the existing CPU gauge I removed because I noticed it wasnt working too.
But I was thinking it could come back using the data thats in the `PICOSTATS` define within `picoTrackerEventManager::MainLoop()` , I think that would work?

**Comment by maks** [comment]
obsolete as of #219
